database/clickhouse/clickhouse_test.go

```diff
@@ -33,12 +33,12 @@ var (
 func clickhouseConnectionString(host, port, engine string) string {
 	if engine != "" {
 		return fmt.Sprintf(
-			"clickhouse://%v:%v?username=user&password=password&database=db&x-multi-statement=true&x-migrations-table-engine=%v&debug=false",
+			"clickhouse://%v:%v/db?username=user&password=password&x-multi-statement=true&x-migrations-table-engine=%v&debug=false",
```

Comment:
this one needed at least until this one merged:
https://github.com/ClickHouse/clickhouse-go/pull/1541

ref: https://github.com/ClickHouse/clickhouse-go/issues/757

other related issue: https://github.com/golang-migrate/migrate/issues/380